package.json

```diff
@@ -85,6 +85,7 @@
     "test:lint": "eslint --cache .",
     "test:main": "npm run build && grunt test",
     "test:cli": "npm run build && bin/qunit.js test/cli/*.js",
+    "build:dev": "node scripts/watch.js",
```

Comment:
This and eslintrc still refer to the old filename. Good to go otherwise!

Reply:
Ooops.